soh/soh/Enhancements/randomizer/location_list.cpp

```diff
@@ -643,6 +644,7 @@ void Rando::StaticData::InitLocationTable() { //
     // Lake Hylia
     locationTable[RC_LH_CHILD_FISHING] =                                               Location::Base(RC_LH_CHILD_FISHING,                                             RCQUEST_BOTH,    RCTYPE_STANDARD,                    RCAREA_LAKE_HYLIA,                   ACTOR_ID_MAX,         SCENE_FISHING_POND,                 0x00,                                              0x3E, "Child Fishing",                               RHT_LH_CHILD_FISHING,                                            RG_PIECE_OF_HEART,                    {},                                                     SpoilerCollectionCheck::RandomizerInf(RAND_INF_CHILD_FISHING),                                                                SpoilerCollectionCheckGroup::GROUP_LAKE_HYLIA,                 true);
     locationTable[RC_LH_ADULT_FISHING] =                                               Location::Base(RC_LH_ADULT_FISHING,                                             RCQUEST_BOTH,    RCTYPE_STANDARD,                    RCAREA_LAKE_HYLIA,                   ACTOR_ID_MAX,         SCENE_FISHING_POND,                 0x00,                                              0x38, "Adult Fishing",                               RHT_LH_ADULT_FISHING,                                            RG_PROGRESSIVE_SCALE,                 {},                                                     SpoilerCollectionCheck::RandomizerInf(RAND_INF_ADULT_FISHING),                                                                SpoilerCollectionCheckGroup::GROUP_LAKE_HYLIA,                 true);
+    locationTable[RC_LH_HYRULE_LOACH] =                                                Location::Base(RC_LH_HYRULE_LOACH,                                              RCQUEST_BOTH,    RCTYPE_STANDARD,                    RCAREA_LAKE_HYLIA,                   ACTOR_ID_MAX,         SCENE_FISHING_POND,                 0x00,                                              0x00, "Hyrule Loach Reward",                         RHT_LH_HYRULE_LOACH,                                             RG_PROGRESSIVE_SCALE,                 {},                                                     SpoilerCollectionCheck::RandomizerInf(RAND_INF_CAUGHT_LOACH),                                                                 SpoilerCollectionCheckGroup::GROUP_LAKE_HYLIA);
```

Comment:
Vanilla item should be a purple rupee not a scale

Reply:
Fixed.